### DISCUSSION TITLE
What determines available PHP versions?

### DISCUSSION THREAD

**Discussion by Llbe** [oerdnj/deb.sury.org#2211] [General]
Hi!

I wonder what determines the currently available PHP versions? How can I know how long PHP 7.4 packages will be available for?

 - I see that all 7.x versions are available for Ubuntu 24.04 (https://launchpad.net/~ondrej/+archive/ubuntu/php/+packages?field.name_filter=7.&field.status_filter=published&field.series_filter=noble)

 - The maintainer is listed as Debian PHP Maintainers

 - They list 7.4 as supported in Bullseye, older PHP versions are listed for EOL versions of Debian (https://wiki.debian.org/PHP#Available_versions) yet they are available in this repo

Probably I'm just missing something here, more info is highly appreciated. Thanks!

**Comment by oerdnj** [comment]
Basically, there’s no guarantee. I will support the packages as long as it feasible. Whatever that means. I can die, I can get bored, I can lose interest, I can get too tired, you see where I am heading.

If using the packages to make money, you can consider getting commercial support. For Freexian PHP offer, we renew the guarantee every year for the next full year.
